pkg/arvo/sys/zuse.hoon

```diff
@@ -5618,6 +5632,9 @@
     ::                                                  ::  ++sa:dejs:format
     ++  sa                                              ::  string as tape
       |=(jon/json ?>(?=({$s *} jon) (trip p.jon)))
+    ::                                                  ::  ++se:dejs:format
+    ++  se                                              ::  string as @tas
+      |=(jon=json ?>  ?=([%s *] jon) (scot %tas p.jon))
```

Comment:
This seems like a bit of a strange operation -- extracting a JSON string, then serializing it as `@tas`. Can you elaborate on your use-case(s)? It seems like it'd be more common to want to parse the extracted string with `+slav`.

Reply:
Use case is simple: I want to go from JSON to a dock, or to an app-name, etc.

https://github.com/urbit/urbit/blob/ff7b2ab7d4f8e4c8daf28b474482b6874301effd/pkg/arvo/lib/invite-json.hoon#L152

Reply:
(note the name is different in that library)

Reply:
Do you have some kind of parsing or validation in mind? What do you want to happen if you're sent JSON that doesn't conform to the "symbol constraint" of `@tas` (ie, lower-case alphanumeric, leading alpha)?

Reply:
@loganallenc, to be clear, this seems wrong regardless of your use-case. I don't see any reason why you would want to serialize inside your (re-)parser here. In the valid case, the serialization is an identity function, and in the invalid case, it's *still* an identity function (see `(scot %tas '1FOO')`). I could see asserting `(sane @tas)`, or sanitizing the input to make it a valid `@tas`.

Reply:
/cc @belisarius222

Reply:
Will switch to using `slav`.